ios/Exponent/Kernel/AppLoader/EXAppLoaderExpoUpdates.m

```diff
@@ -37,6 +37,7 @@ @interface EXAppLoaderExpoUpdates ()
 @property (nonatomic, assign) EXAppLoaderRemoteUpdateStatus remoteUpdateStatus;
 @property (nonatomic, assign) BOOL shouldShowRemoteUpdateStatus;
 @property (nonatomic, assign) BOOL isUpToDate;
+@property (nonatomic, assign) BOOL isLoadingDevelopmentJavaScriptResource;
```

Comment:
Do you mind adding here some comment as well?
I mean, when I navigate throughout the code I usually end in the property/variable declaration and wonder why it's there. I know that there's a comment below in the code, but it would be harder to spot it at the first glace.